### PR TITLE
Exiv2 v27_4 RC1 (effort2)

### PR DESCRIPTION
**Pull request by clanmills**

### CODE REVIEW

File: releasenotes/releasenotes.txt [hunk 1]
```diff
@@ -1,5 +1,72 @@
...
+| Christoph    | Nikon and Canon Support                           |
+| David        | Help with legal investigation concerning bmff     |
+| Freddie      | Fuji support                                      |
+| Leo          | New python test scripts                           |
+| Milos        | DNG, easyaccess support                           |
+| Peter K      | bmff code                                         |
+| Peter S      | langAltValue.read() fix and helpful issue reports |
+
+1) Bug and security fixes:
+   - bmff support (.CR3, .HEIC and .AVIF).
```
Comment (kmilos) [inline releasenotes/releasenotes.txt:23]: Maybe we should be explicit it is read support only?

Comment (clanmills) [reply releasenotes/releasenotes.txt:23]: Yes.  I could explicitly say that.  As you know, that detail is documented in man/man1/exiv2.1

At the start of the bmff project, the design was to add code to jp2image.cpp for kinds of other bmff files.  For practical reasons, Peter opened bmffimage.cpp.  I was looking at the man page this morning and thought "Ah yes, we have read-write support for JP2, and read-only for the new formats.".  So, it's probably better and simpler to have them in different image handlers.

Comment (clanmills) [reply releasenotes/releasenotes.txt]: https://github.com/Exiv2/exiv2/pull/1498/commits/08b5eb6136b999603b8dd340276fd214eb62bd7a

**Review by kmilos — APPROVED** [review]

**Comment by clanmills** [comment]
Great Team Work.  Exiv2 v0.27.4 RC1 is going out the door in 60 minutes.

**Pull request merged**